docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -30,10 +30,12 @@ The ``uw`` mode for handling configs.
 ``compare``
 -----------
 
-.. code:: sh
+.. code-block:: text
 
   $ uw config compare --help
-  usage: uw config compare --file-1-path PATH --file-2-path PATH [-h] [--file-1-format {ini,nml,sh,yaml}] [--file-2-format {ini,nml,sh,yaml}] [--quiet] [--verbose]
+  usage: uw config compare --file-1-path PATH --file-2-path PATH [-h]
+                           [--file-1-format {ini,nml,sh,yaml}] [--file-2-format {ini,nml,sh,yaml}]
+                           [--quiet] [--verbose]
```

Comment:
Following a new doc guideline, I captured `--help` output in a terminal set at 100 columns so that `argparse` could wrap and indent its synopsis line in a consistent way for all examples. I wan't aware until recently that `argparse` does this automatic wrapping and indenting.